.gitignore

```diff
@@ -27,13 +27,13 @@ kubernetes.github.io.iml
 nohup.out
 
 # Hugo output
-public/
-resources/
+/public/
+/resources/
 .hugo_build.lock
 
 # Netlify Functions build output
 package-lock.json
-functions/
+/functions/
 node_modules/
```

Comment:
Optional extra fix: also anchor `node_modules`

Reply:
@sftim thanks for the review, and I have fixed it and squashed the commits! :smiley: